client/src/App.jsx

```diff
@@ -12,23 +12,34 @@ import PrivateRoute from "./routes/PrivateRoute";
 import Dashboard from "./containers/Dashboard";
 import Course from "./containers/Course/Course";
 import Navbar from "./containers/Navbar/Navbar";
+import MatchableJoin from "./containers/Matchable/MatchableJoin";
+import MatchableCreate from "./containers/Matchable/MatchableCreate";
 import theme from "./theme";
 import { menu_options } from './utils/constants';
 
+const Container = styled.div`
```

Comment:
it is better to move all styled component into a `components` folder, we can have a components fooder in the root level

Reply:
But we already have a `components` folder at the root level containing shared components.
What should we do in this case?

Reply:
Done!